dist/gmusic.js

```diff
@@ -185,7 +185,8 @@ proto.playback = {
 
   // Time functions
   getPlaybackTime: function () {
-    return parseInt(this.playback._sliderEl.getAttribute('aria-valuenow'), 10);
+    // In milliseconds
+    return this.playback._sliderEl.value;
```

Comment:
Not effected by i18n in this case, also is an integer:

![image](https://cloud.githubusercontent.com/assets/760989/15630932/4b91e968-251e-11e6-96f5-675586c7073f.png)